modules/http2/src/test/java/org/glassfish/grizzly/http2/AbstractHttp2Test.java

```diff
@@ -45,14 +46,23 @@
 
 /**
  * General HTTP2 client/server init code.
- * 
+ *
  * @author Alexey Stashok
  */
 public abstract class AbstractHttp2Test {
+    static {
```

Comment:
Logging in tests, so you can see order of method executions in multithreaded tests.